ConvivaTestApp/src/main/java/com/bitmovin/analytics/conviva/testapp/MainActivity.kt

```diff
@@ -27,31 +27,31 @@ class MainActivity : AppCompatActivity() {
         val intent = intent
         val autoPlay = intent.getBooleanExtra(AUTOPLAY_KEY, false)
         var vmapTagUrl = intent.getStringExtra(VMAP_KEY)
-        bitmovinPlaybackConfiguration = PlaybackConfiguration()
-        bitmovinPlaybackConfiguration.isAutoplayEnabled = autoPlay
+        bitmovinPlaybackConfig = PlaybackConfig()
+        bitmovinPlaybackConfig!!.isAutoplayEnabled = autoPlay
 
         // Creating a new PlayerConfiguration
-        bitmovinPlayerConfiguration = PlayerConfiguration()
-        // Assign created SourceConfiguration to the PlayerConfiguration
-        bitmovinPlayerConfiguration.playbackConfiguration = bitmovinPlaybackConfiguration
+        bitmovinPlayerConfig = PlayerConfig()
+        bitmovinPlayerConfig!!.playbackConfig = bitmovinPlaybackConfig!!
```

Comment:
The force unwrap here should not be needed.

Even better would be:

```suggestion
        bitmovinPlayerConfig = PlayerConfig().apply {
        	playbackConfig = bitmovinPlaybackConfig
        }
```

Reply:
updated